core/src/main/java/de/jplag/reporting/reportobject/model/SubmissionFile.java

```diff
@@ -1,4 +1,9 @@
 package de.jplag.reporting.reportobject.model;
 
+/**
+ * Represents metadata about a submission file used in a plagiarism comparison.
+ * @param tokenCount The number of tokens contained in the file. This indicates the file's size or complexity in terms
```

Comment:
I would remove the second sentence
```suggestion
 * @param tokenCount The number of tokens contained in the file.
```